packages/modals-collection/src/apps/quick-add-app/quick-add-app-modal.tsx

```diff
@@ -8,7 +8,7 @@ import { useI18n, useScopedI18n } from "@homarr/translation/client";
 import type { validation } from "@homarr/validation";
 
 interface QuickAddAppModalProps {
-  onClose: (createdAppId: string) => Promise<void>;
+  onClose: (createdAppId: string) => void;
```

Comment:
I would suggest that we still allow a `Promise` here. You could use the type `MaybePromise` which allows both promise and no promise values